heman/api/infoenergia/__init__.py

```diff
@@ -0,0 +1,51 @@
+from datetime import datetime
+from dateutil.relativedelta import relativedelta
+import json
+import time
+
+from flask import current_app, jsonify, request, Response
+from pymongo import ASCENDING
+
```

Comment:
`datetime`, `relativedelta`, `time`, `request` and `ASCENDING` imports are not used :fearful: :laughing: :laughing: